tests/unit/file_utils.py

```diff
@@ -0,0 +1,380 @@
+# Copyright (C) 2020 NRL
+# Author: Angeline Burrell
+
+import bz2
+import collections
+import copy
+from glob import glob
+import numpy as np
+import os
+import unittest
+
+import pydarnio
+
+import dmap_data_sets
+import fitacf_data_sets
+import grid_data_sets
+import iqdat_data_sets
+import map_data_sets
+import rawacf_data_sets
+
+
+def get_test_files(test_file_type, test_dir=os.path.join("..", "testfiles")):
+    """ Generate a dictionary containing the test filenames
+
+    Parameters
+    ----------
+    test_file_type : str
+        Accepts 'good', 'stream', 'empty', and 'corrupt', along with
+        'borealis-vXX_' or 'borealis-vXX-site_' as a prefix to these options
+    test_dir : str
+        Directory containing the test files
+        (default=os.path.join('..', 'testfiles'))
+
+    Returns
+    -------
+    test_files : list or dict
+        Dict of good files with keys pertaining to the file type, a list
+        of corrupt files, or a list of stream files
+
+    """
+    # Ensure the test file type is lowercase
+    test_file_type = test_file_type.lower()
+
+    # See if this is a borealis test
+    test_subdir = test_file_type.split('_')
+    if len(test_subdir) > 1:
+        # Only the last bit specifies test_file_type
+        test_file_type = test_subdir[-1]
+
+        # Keep the first part of the specifier intact and extend the
+        # test directory
+        test_dir = os.path.join(test_dir, '_'.join(test_subdir[:-1]))
+
+    # Get a list of the available test files
+    files = glob("ls {:s}".format(os.path.join(test_dir, test_file_type, "*")))
```

Comment:
does having the 'ls' in there for this glob work for the other tests? I could not get this to work for me for the Borealis tests. I would suggest the following but want to be sure it doesn't affect the other tests. 
```suggestion
    files = glob("{:s}".format(os.path.join(test_dir, test_file_type, "*")))
```